receiver/jmxreceiver/receiver_test.go

```diff
@@ -97,6 +106,29 @@ otel.metrics.exporter = otlp
 otel.resource.attributes = abc=123,one=two`,
 			"",
 		},
+		{
+			"handles password file",
+			&Config{
+				Endpoint:     "myhost:12345",
+				TargetSystem: "mytargetsystem",
+				OTLPExporterConfig: otlpExporterConfig{
+					Endpoint: "https://myotlpendpoint",
+				},
+				Username:     "myusername",
+				PasswordFile: passwordFilePath,
+			},
+			`javax.net.ssl.keyStorePassword = keypass
+javax.net.ssl.trustStorePassword = trustpass
+otel.exporter.otlp.endpoint = https://myotlpendpoint
+otel.exporter.otlp.timeout = 0
+otel.jmx.interval.milliseconds = 0
+otel.jmx.password = mypassword
```

Comment:
Should we change the otel jar to be able to use a password file?

Reply:
We could, but the temp file is already read-restricted to owner-only access, so if the collector runs as root, it should be fine. https://cs.opensource.google/go/go/+/refs/tags/go1.22.0:src/os/tempfile.go;l=47